### ISSUE TITLE
Can't run VOT using the 0.1.16 version of attributee

### ISSUE THREAD

**Issue by nvbln** [votchallenge/toolkit#42]
VOT does not work with the newest version of attributee (0.1.6). Instead, downgrading to 0.1.5 works.

**Comment by lukacu** [comment]
Thanks for the report, I will fix it shortly.

**Comment by Seerkfang** [comment]
I met the same problems and I find that the question is: in vot/experiment::transformers = List(Object(transformer_resolver), default=[]) the default is not none, thus attibutee0.1.6/Attribute::__init__() will execute the "else...else" choice when meeting self._default = default if is_undefined(default) else (None if default is None else self.coerce(default, {})), which is reloaded in containers::self.readonly. This function would turn to the readonly function in __init__.py, and the self._readonly hasn't been assigned a value yet (two lines after the assignment of self._default in __init__.py)!!!

**Comment by lukacu** [comment]
Yes, I have a fix in repository, but did not manage to release a new version. You can install it from github directly or install 0.1.5

**Comment by lukacu** [comment]
New version of library available on PyPi, closing the issue.
